contexts/AuthzContext.tsx

```diff
@@ -121,7 +121,7 @@ export function AuthzContextProvider({children}: {children: React.ReactNode}) {
     proposer: accounts[fcl.withPrefix(voucher.proposalKey.address)],
```

Comment:
First of all, thank you for submitting this PR! We've encountered this issue while working with multi-sig transactions where one of the accounts is not originated in Dev Wallet.

Both `proposer` and `payer` also need to be updated the same way as you did for authorizers in line 124.